modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -259,20 +259,10 @@ def update_status(status:, code: nil, detail: nil, raise_on_error: false)
           }.deep_stringify_keys
         )
 
-        return if auth_headers.blank? # Go no further if we've removed PII
+        return if form_data.blank? # Go no further if we've removed PII
 
         if status == 'submitted' && email_present?
-          AppealsApi::AppealReceivedJob.perform_async(
-            {
-              receipt_event: 'nod_received',
-              email_identifier:,
-              first_name: veteran_first_name,
-              date_submitted: veterans_local_time.iso8601,
-              guid: id,
-              claimant_email: claimant.email,
-              claimant_first_name: claimant.first_name
-            }.deep_stringify_keys
-          )
+          AppealsApi::AppealReceivedJob.perform_async(id, self.class.name, appellant_local_time.iso8601)
```

Comment:
These `update_status` methods in the models are the only place where this job is kicked off